server/balancer.go

```diff
@@ -138,15 +139,15 @@ func (l *balanceLeaderScheduler) Schedule(cluster *clusterInfo) Operator {
 type balanceRegionScheduler struct {
 	opt      *scheduleOption
 	rep      *Replication
-	cache    *idCache
+	cache    *cache.IDTTL
```

Comment:
The name IDTTL is wired, please figure out a better name

Reply:
Do you have any suggestion? @ngaut

Reply:
CacheWithTTL

Reply:
CacheWithTTL

Reply:
CacheWithTTL

Reply:
Can't use name starts with 'Cache', because the package is 'cache', lint tools will complain about it.

Reply:
TTLUint64?